sklearn/linear_model/_coordinate_descent.py

```diff
@@ -1181,6 +1181,11 @@ def _get_estimator(self):
     def _is_multitask(self):
         """Bool indicating if class is meant for multidimensional target."""
 
+    @staticmethod
+    @abstractmethod
+    def path():
```

Comment:
To communicate some structure for the input:

```suggestion
    def path(X, y, **kwargs):
```

(`enet_path` and `lasso_path` do not have the same `**kwargs**)